packages/core/src/parsers/colors/colors.ts

```diff
@@ -38,6 +40,14 @@ export function parseColor({
     }
   }
 
+  console.log(value);
+
+  const color = theme.getValue('colors', value as Color);
+
+  if (color && color.indexOf('$') === 0) {
+    return placeholderParser({ value, property, color });
```

Comment:
Isn't it enough to call recursively the color parser:

```typescript
if (color && color.indexOf('$') === 0) {
   return parseColor({ value: color.substring(1), property });
}
```

Reply:
in this way we are missing the condition that check if the original color is in place

Reply:
right! nice catch!